pkgs/development/tools/packer/default.nix

```diff
@@ -40,7 +44,178 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir -p $out/bin
-    mv bin/* $out/bin
+    mv bin/* $out/bin # */
+  '';
+
+  passthru.updateScript = writeScript "update-packer" ''
+    #!${bash}/bin/bash
```

Comment:
did you try to run [`make deps`](https://github.com/mitchellh/packer/blob/master/Makefile#L30) in packer source before `go2nix save`?